.github/workflows/pull-request.yml

```
@@ -168,6 +171,27 @@
             ISSUE_GUIDELINES.md
             PR_GUIDELINES.md
             README.md
+          )
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)